app/models/project.rb

```diff
@@ -1,10 +1,14 @@
 class Project < ActiveRecord::Base
+  before_create :set_github_repo
   belongs_to :organization
   acts_as_ordered_taggable
   acts_as_ordered_taggable_on :technologies, :causes
 
-  attr_accessible :organization_id, :name, :url, :github_repo, :description, :notes, :cause_list, :technology_list, :is_active, :is_approved
-  validates_presence_of :name, :github_repo
+  attr_accessor :submitted_github_url
+  attr_accessible :organization_id, :name, :url, :github_repo, :description, :notes, :cause_list, :technology_list, :is_active, :is_approved, :submitted_github_url
+  validates_presence_of :name
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.